common/persistence/sql/sqlplugin/sqlite/plugin.go

```diff
@@ -177,7 +179,14 @@ func buildDSNAttr(cfg *config.SQL) (url.Values, error) {
 				key, value,
 			)
 		}
-		parameters.Set(key, value)
+
+		if strings.Contains(queryParameters, key) {
```

Comment:
I think we need exact match for the keys, otherwise if the key contains those value it would be a problem, for example key="cache_mode".

Reply:
`cache_mode` would return false and would fall through and likely become an error because that pragma wouldn't be found.

now `cach` would match and just simply pass through, but not have the desired effect.

I can change this.